### PR TITLE
DBVIS-10096 Try to improve performance by growing by size doubling.

### PR DESCRIPTION
**Pull request by robert-dbvis**

### CODE REVIEW

File: RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java [hunk 1]
```diff
@@ -36,28 +36,23 @@ class DefaultTokenFactory implements TokenFactory {
 	private int currentFreeToken;
 
 	protected static final int DEFAULT_START_SIZE	= 30;
-	protected static final int DEFAULT_INCREMENT		= 10;
-
 
 	/**
 	 * Constructor.
 	 */
 	DefaultTokenFactory() {
```
Comment (ptorngren) [inline RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java:43]: The change should be tagged with the JIRA

File: RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java [hunk 2]
```diff
@@ -36,28 +36,23 @@ class DefaultTokenFactory implements TokenFactory {
...
 
 	/**
 	 * Constructor.
 	 *
 	 * @param size The initial number of tokens in this factory.
-	 * @param increment How many tokens to increment by when the stack gets
-	 *        empty.
 	 */
-	DefaultTokenFactory(int size, int increment) {
+	DefaultTokenFactory(int size) {
```
Comment (ptorngren) [inline RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java:54]: The change should be tagged with the JIRA

File: RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java [hunk 3]
```diff
@@ -74,13 +69,14 @@ class DefaultTokenFactory implements TokenFactory {
 	 * request is made and no more tokens are available.
 	 */
 	private void augmentTokenList() {
```
Comment (ptorngren) [inline RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java:72]: The change should be tagged with the JIRA

**Review by ptorngren — CHANGES_REQUESTED** [review]
1. The changes should be tagged with the JIRA
2. DefaultTokenFactory.increment is unused, should be deleted

File: RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java (hunk 2 above)
Comment (robert-dbvis) [reply RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java:54]: Done now.

File: RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java (hunk 3 above)
Comment (robert-dbvis) [reply RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java:72]: Done now

File: RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java (hunk 1 above)
Comment (robert-dbvis) [reply RSyntaxTextArea/src/main/java/org/fife/ui/rsyntaxtextarea/DefaultTokenFactory.java:43]: Done now.

**Review by ptorngren — APPROVED** [review]
Looks good!

**Pull request merged**